.github/workflows/release-layer-nodejs.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
-          node-version: 14
+          node-version: 16
 
       - name: Build
         run: |
```

Comment:
Do you want to add Node 20 to the runtimes list on line 74?

Reply:
Yeah, thanks for catching it :) Added